Find a functionally equivalent code:android.os.Process.myTid:COMMENT
Method Modifier: public      static      final       
Comment:/**
 * Returns the identifier of the calling thread, which be used with
 * {@link #setThreadPriority(int, int)}.
 */

@@ -1,3 +1,3 @@
 {
-    return Libcore.os.gettid();
+    return Os.gettid();
 }

